JavaScript_02/JavaScript_02.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="bg">
+	<head>
+		<meta charset="UTF-8"/>
+		<title>JAVA_01</title>
+		<script src="JavaScript_02.js"></script>
+		
+		<style>
+			body
+			{
+				border: 2px solid black;
+				text-align: center;
+			}
+		</style>
+	</head>
+	<body>
+		<h1 style="text-align:center;"> LOGIN </h1>
+		
+		<form method="post" onsubmit="return false">
+			<label for="Username">Потребителско име: </label>
+			<input type="text" id="Username" name="username"><br>
+			<label id="invalidUsername"></label><br>
+			
+			<label for="Password">Парола: </label>
+			<input type="password" id="Password" name="passwd"><br>
+			<label id="invalidPassword"></label><br>
+			
+			<label for="Confirm_Password">Повтори паролата: </label>
+			<input type="password" id="Confirm_Password" name="password_conf"><br>
+			<label id="invalidConfirmPassword"></label><br>
+				
+			<input type="submit" value="Submit" onclick="validate()">
```

Comment:
Maybe you may want to rename the function. What does the function do? Is it only validating or it is doing also something else additionally now?